cluster/gce/gci/configure-helper-common.sh

```diff
@@ -1367,7 +1367,19 @@ function prepare-log-file {
 function prepare-kube-proxy-manifest-variables {
   local -r src_file=$1;
 
-  local -r kubeconfig="--kubeconfig=/var/lib/kube-proxy/kubeconfig"
+  local kubeconfig="--kubeconfig=/var/lib/kube-proxy/kubeconfig"
+  if [[ "${ARKTOS_SCALEOUT_SERVER_TYPE:-}" == "node" ]]; then
+    echo "DBG:Set tenant-server-kubeconfig parameters:  ${TENANT_SERVER_KUBECONFIGS}"
+    kubeconfig+=" --tenant-server-kubeconfig=${TENANT_SERVER_KUBECONFIGS}"
+  fi
+
+  local proxy_tpconfig_mount=""
+  local proxy_tpconfig_volume=""
+  if [[ "${ARKTOS_SCALEOUT_SERVER_TYPE:-}" == "node" ]]; then
```

Comment:
i recall the kube-proxy is required to be run on TP master as well. so the var/lib/kube-proxy has already been mounted into the kube-proxy container? or it was started differently on TP master node. ?

Reply:
kube-proxy on TP/RP-master seem to connect to proper TP-master already. this code change only care about RP nodes.